### ISSUE TITLE
Ctrl-D => undefined method `encode' for nil:NilClass

### ISSUE THREAD

**Issue by fxkr** [gogotanaka/hilbert#65]
Ctrl-D should stop "hilbert -i". Instead:

```
Enjoy! -> undefined method `encode' for nil:NilClass
```

**Comment by fxkr** [comment]
Versions:

```
% gem install --user hilbert
Fetching: dydx-0.2.7000001.gem (100%)
Successfully installed dydx-0.2.7000001
Fetching: hilbert-0.0.2700210.gem (100%)
Building native extensions.  This could take a while...
Successfully installed hilbert-0.0.2700210
Parsing documentation for dydx-0.2.7000001
Installing ri documentation for dydx-0.2.7000001
Parsing documentation for hilbert-0.0.2700210
Installing ri documentation for hilbert-0.0.2700210
Done installing documentation for dydx, hilbert after 1 seconds
2 gems installed
```

**Comment by gogotanaka** [comment]
@fxkr good proposal. I'm gonna fix it.

**Comment by yonkeltron** [comment]
Can confirm this happens with current version (0.0.2700300) on OSX Yosemite.

```
$ ruby --version
ruby 2.1.4p265 (2014-10-27 revision 48166) [x86_64-darwin14.0]
$ uname -a
Darwin Concurrent-Chickpea.local 14.0.0 Darwin Kernel Version 14.0.0: Fri Sep 19 00:26:44 PDT 2014; root:xnu-2782.1.97~2/RELEASE_X86_64 x86_64
```

**Comment by gogotanaka** [comment]
@fxkr @yonkeltron I'm so sorry for my delay.

I've fixed this issue at https://github.com/gogotanaka/Hilbert/commit/631bd6e5191cf7675c52d64c63314e4c9f5b82e9. 

and released as v0.0.2700410. 

If any issues still remains, please feel free to ask me. thanks.
